### ISSUE TITLE
Survey task steals keyboard focus

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#3298]
## Package
lib-classifier

## Describe the bug
The choice buttons in the survey task use `autofocus`, leading to a couple of serious usability and accessibility problems:
1. The filter component is skipped, meaning that users of screen readers or screen magnifiers might not even know that the list of choices can be filtered.
2. When the popup tutorial is open, keyboard focus is underneath the modal layer.
<img width="840" alt="Screenshot of Wildcam Darién. The modal tutorial is open but the Ñeque option in the Task panel has keyboard focus." src="https://user-images.githubusercontent.com/59547/173299561-b4140203-6ae5-42e1-a07c-69458993d5c0.png">

## To Reproduce
I think this can be reproduced on any survey task project. You may need to be logged out, in order to force the tutorial to open.

## Expected behavior
When the tutorial is open, it should have focus. It's already coded to do this, so avoid having autofocus anywhere else in the classifier when the tutorial is open.

If the Task panel must be focused (eg. when starting a new classification), keyboard focus should go to the first focusable element in the panel, which is the filter control. In that case, I'm not sure how a user that relies on a screen reader, or a screen magnifier, would know that the subject has changed but I can see how focusing the filter control would make classification easier for a keyboard user.

## Additional context
- [WCAG 2.4.3: Focus order.](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
- [WCAG 3.2.1: On focus.](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html)

Also see #2455, where autofocus skips survey task choice descriptions for screen reader users.
